timeseries/src/autogluon/timeseries/models/gluonts/mx/models.py

```diff
@@ -455,7 +468,7 @@ def train(self, train_data, validation_data=None, **kwargs):
         return self.predictor
 
 
-class ProphetModel(AbstractGluonTSModel):
+class ProphetModel(AbstractGluonTSMXNetModel):
```

Comment:
I guess Prophet doesn't really depend on MXNet, but I'm fine with keeping it in this file.

On a related note, should we even enable this model / expose it in the presets given that it's not tested properly?

Reply:
👍 commenting it out from presets.